Neos.ContentRepositoryRegistry/Classes/Service/EventMigrationService.php

```diff
@@ -222,6 +231,88 @@ public function migratePropertiesToUnset(\Closure $outputFn)
         }
     }
 
+
+    /**
+     * Adds a dummy workspace name to the events meta-data, so it can be rebased
+     *
+     * The value of the payload for `workspaceName` is only required to successfully instantiate a command by its metadata.
+     * This is only necessary for rebasing where directly override the workspace name to the target one.
+     * Thus, we simply enter a dummy string "missing:{contentStreamId}".
+     *
+     * Needed for #4708: https://github.com/neos/neos-development-collection/pull/4708
+     *
+     * Included in March 2024 - before final Neos 9.0 release
+     *
+     * @param \Closure $outputFn
+     * @return void
+     */
+    public function migrateMetaDataToWorkspaceName(\Closure $outputFn)
+    {
+        $this->eventsModified = [];
+
+        $backupEventTableName = DoctrineEventStoreFactory::databaseTableName($this->contentRepositoryId)
+            . '_bak_' . date('Y_m_d_H_i_s');
```

Comment:
another nitpicky suggestion:
```suggestion
            . '_bkp_' . date('Y_m_d_H_i_s');
```

Reply:
followed the pattern of another migration, please adjust also there if here ^^ // cc @ahaeslich